modin/experimental/engines/pyarrow_on_ray/frame/data.py

```diff
@@ -14,17 +14,17 @@
 import pandas
 from pandas.core.dtypes.cast import find_common_type
 
-from .partition_manager import PyarrowOnRayFrameManager
-from modin.engines.base.frame.data import BasePandasFrame
+from .partition_manager import PyarrowOnRayFramePartitionManager
+from modin.engines.base.frame.data import PandasFrame
 
 import ray
 
 
-class PyarrowOnRayFrame(BasePandasFrame):
+class PyarrowOnRayFrame(PandasFrame):
 
-    _frame_mgr_cls = PyarrowOnRayFrameManager
+    _partition_mgr_cls = PyarrowOnRayFramePartitionManager
 
-    def _apply_index_objs(self, axis=None):
+    def synchronize_labels(self, axis=None):
         """Lazily applies the index object (Index or Columns) to the partitions.
```

Comment:
this summary probably needs updating, too

Reply:
Updated